intel_extension_for_transformers/transformers/modeling/modeling_auto.py

```diff
@@ -137,7 +137,7 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
                 if quantization_config is None:
                     if use_llm_runtime:
                         quantization_config = WeightOnlyQuantConfig(
-                            compute_dtype="int8", weight_dtype="int8"
+                            compute_dtype="bf16", weight_dtype="int8"

```

Comment:
@kevinintel 
Hi Kevin ge
If we default use bf16 for load in 8bit, which mean it can only run in SPR, otherwise it may have problem

Reply:
if running on no-spr machine, bf16 will fall back to fp32, from yu